<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Fareed-PC" ignored="4" name="Default test(failed)" tests="2" failures="1" timestamp="18 Jul 2020 03:37:47 GMT" time="0.078" errors="0">
  <testcase name="signIn" time="0.02" classname="tests.TestNGConcepts">
    <failure type="java.lang.AssertionError" message="expected [Test] but found [school]">
      <![CDATA[java.lang.AssertionError: expected [Test] but found [school]
at tests.TestNGConcepts.signIn(TestNGConcepts.java:13)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- signIn -->
  <testcase name="homePage" time="0.001" classname="tests.TestNGConcepts">
    <skipped/>
  </testcase> <!-- homePage -->
  <testcase name="checkTitle" time="0.0" classname="tests.TestNGConcepts">
    <ignored/>
  </testcase> <!-- checkTitle -->
  <testcase name="customerPage" time="0.0" classname="tests.TestNGConcepts">
    <ignored/>
  </testcase> <!-- customerPage -->
  <testcase name="logout" time="0.0" classname="tests.TestNGConcepts">
    <ignored/>
  </testcase> <!-- logout -->
  <testcase name="dashboard" time="0.0" classname="tests.TestNGConcepts">
    <ignored/>
  </testcase> <!-- dashboard -->
</testsuite> <!-- Default test(failed) -->
